### DISCUSSION TITLE
Link preset download

### DISCUSSION THREAD

**Discussion by Horea-Danci-Personal** [matteobruni/tsparticles#5246] [Q&A]
Hello! Where can I export the Links preset from? I can't seem to find it. Thnak you!

**Comment by matteobruni** [comment]
> Hello! Where can I export the Links preset from? I can't seem to find it. Thnak you!

I don’t understand what do you mean by downloading it, but this is the presets repository: https://github.com/tsparticles/presets/tree/main/presets
